walkthroughs/rhte-2019-hackathon/walkthrough.adoc

```diff
@@ -567,4 +567,214 @@ Details of table names and schemas were provided in the previous section - *API
 
 
 [time=90]
-== Front end visualisation using React.js and Patternfly.js
+== Front end visualisation using React and Patternfly 4
+
+In this section you will use Patternfly's React components to create a web
+application to display the latest parking meter and junction traffic
+information. PatternFly is an open source design system created to enable
+consistency and usability across a wide range of applications and use cases.
+
+For the Parking Meters page you will:
+
+. Display a Google Map
+. Render a map marker for each meter
+. Cluster the meters when the map is zoomed out
+. Support filters for each meter state, e.g show only "available" parking meters
+
+{empty} +
+
+For the Traffic Junctions page you will:
+
+. Display a Google Map
+. Use Heatmap functionality from Google Maps to render traffic conditions
+
+{empty} +
+
+The end result will look similar to the images below. Here's an example of the
+Parking Meters being displayed:
+
+{empty} +
+
+image::images/ui-parking-meters.png[role="integr8ly-img-responsive"]
+
+{empty} +
+
+And here's the traffic heatmap built using the junction data
+
+{empty} +
+
+image::images/ui-traffic-heatmap.png[role="integr8ly-img-responsive"]
+
+=== Create the UI GitHub Repository & Import it into CodeReady Workspaces
+
+. Open the UI template repostory on GitHub at link:https://github.com/evanshortiss/rhte-2019-hackathon-on-rhmi-template-ui[evanshortiss/rhte-2019-hackathon-on-rhmi-template-ui].
+. Click the green *Use this template* button.
+.. Select an *Owner* organisation and enter a *Repository name*.
+.. Click *Create repository from template* button.
+. Open link:{che-url}[CodeReady Workspaces]. The *New Workspace* wizard will be displayed.
+.. Enter a name such as *RHTE Hackaton Frontend*.
+.. Select *Node10* in the *Select stack* section.
+.. In the *Projects* section click *Add or Import Project*
+.. Choose *GitHub*, and authenticate if necessary by clicking the *Connect your github account* button.
+.. Once your accounts and organisations find the repository you created a few moments ago and check the box next to it in the list.
```

Comment:
Something not quite right with the flow of this sentence.

Reply:
Absolutely. Fixed it 👍